cookbooks/fb_helpers/libraries/fb_helpers.rb

```diff
@@ -92,14 +92,14 @@ def self.commentify(comment, arghash = {})
       curr_line = start
 
       # For multi-line comments, line-break right after initial start tag
-      if finish != ''
-        final_comment += "#{curr_line}\n"
-        curr_line = ''
-        prefix = ''
-      else
+      if finish == ''
```

Comment:
in case you're wondering this rule is basically, positive case first, for readability